app/preprints/-components/preprint-file-render/template.hbs

```diff
@@ -0,0 +1,50 @@
+<div local-class='file-renderer-container {{if this.isMobile 'mobile'}}'>
+    <div local-class='file-container'>
+        <FileRenderer
+            @allowCommenting={{this.allowCommenting}}
```

Comment:
It seems like `FileRenderer` isn't doing anything with this new parameter here. It looks like it was used to [determine if we needed to do a `postMessage`](https://github.com/CenterForOpenScience/ember-osf/blob/7835594c7f3e7972ca3307148800ed1848cfe444/addon/components/file-renderer/component.js#L59)?

Reply:
We can address it in a separate JIRA ticket/PR if it doesn't seem easy to implement. I'll leave that call up to you

Reply:
I don't think the `allowCommenting` parameter is being used to enable Hypothesis though. I'm not the most familiar with how Hypothesis works, but in ember-osf, there is some `window.postMessage` being done based on `allowCommenting`. I'll reach out to others who may be more familiar with how this works to see what needs to be done